iphone/Classes/TiUIWebView.m

```diff
@@ -715,8 +715,8 @@ -(CGFloat)contentHeightForWidth:(CGFloat)value
     CGRect oldBounds = [[self webview] bounds];
     BOOL oldVal = webview.scalesPageToFit;
     [webview setScalesPageToFit:NO];
-    [webview setBounds:CGRectMake(0, 0, 10, 1)];
-    CGFloat ret = [webview sizeThatFits:CGSizeMake(10, 1)].height;
+    [webview setBounds:CGRectMake(0, 0, value, 1)];
```

Comment:
@hansemannn  That value has been made 10 for ticket TIMOB-11777 .I verified the rotation as mentioned in TIMOB-117777, it is working fine  with this change as well. I couldn't get exact test case of TIMOB-11777 as this is very old ticket. If you can get exact test case we can verify same.